src/applications/vaos/routes.jsx

```diff
@@ -24,6 +24,7 @@ const routes = (
     />
     <Route path="appointments/pending" component={PendingAppointmentsPage} />
     <Route path="appointments/pending/:id" component={PendingAppointmentPage} />
+    <Route path="appointments/confirmed" component={ConfirmedAppointmentPage} />
```

Comment:
After fixing imports, this should be:
```
 <Route path="appointments/confirmed" component={ConfirmedAppointmentPage} />
```

Note the change in the component with the "s"

Reply:
I'm confused about the 2 pages. Is the ConfirmedAppointmentPage replacing the ConfirmedAppontmentsPage (I created this one)?

Reply:
The singular one is the detail page that I worked on.  The one you are adding is the list. The singular one should probably be renamed `ConfirmedAppointmentDetailsPage` for more clarity.

Reply:
Looks like we are doing duplicate work. I created components/ConfirmedAppointment which is similar to the ConfirmedAppointmentPage. The component is supposed to represent the list of confirmed appointments.